Classes/Encoder/UrlEncoder.php

```diff
@@ -1128,7 +1128,7 @@ protected function initializeConfiguration() {
 	protected function isLinkingAcrossDomains() {
 		$result = true;
 
-		foreach (array_reverse($this->tsfe->rootLine) as $page) {
+		foreach ($this->tsfe->rootLine as $page) {
```

Comment:
Removing `array_reverse` would allow crossdomain detection to work with nested domains.